### PR TITLE
Softening introductions on several pages

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR adds some explanation as to why you'd choose Office Scripts, as well as adding more independence to the doc set by broadening the audience slightly.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a00027b](https://github.com/OfficeDev/office-scripts-docs/commits/a00027b5d5aa37e3317794ede97510e023a96f49)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Issue_Template.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/Issue_Template.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/README.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/index.md) | :white_check_mark:Succeeded | |
[docs/overview/code-editor-environment.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/code-editor-environment.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/testing/troubleshooting.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/testing/troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131849199473-56%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gHF9XX21Rj5l0XQpveVqw067cT%2FJIoMnxpX45n5RPlA%3D&st=2019-12-13T18%3A45%3A28Z&se=2020-01-13T18%3A50%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/excel.md [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
 ![A gif showing a script being recorded and replayed on different worksheets](../images/teaser-animation.gif)
 
 [!INCLUDE [Preview note](../includes/preview-note.md)]
 
-## Action Recorder
+## When to use Office Scripts
+
+Scripts allow you to record and replay your Excel actions on different workbooks and worksheets. If you find yourself doing the same things over and over again, an Office Script can help you by reducing your whole workflow to a single button press.
 
+As an example, say you start your work day by opening a .csv file from an accounting site in Excel. You then spend several minutes deleting unnecessary columns, formating a table, adding formulas, and creating a PivotTable in a new worksheet. Those action you repeat daily can be recorded once with the Action Recorder. For then on, running the script will take care of your entire .csv conversion. You'll not only remove the risk of forgetting steps, but be able to share your process with others without having to teach them anything. Office Scripts automate your common tasks so you and your workplace can be more efficient and productive.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/excel.md]:
```suggestion
As an example, say you start your work day by opening a .csv file from an accounting site in Excel. You then spend several minutes deleting unnecessary columns, formatting a table, adding formulas, and creating a PivotTable in a new worksheet. Those actions you repeat daily can be recorded once with the Action Recorder. For then on, running the script will take care of your entire .csv conversion. You'll not only remove the risk of forgetting steps, but be able to share your process with others without having to teach them anything. Office Scripts automate your common tasks so you and your workplace can be more efficient and productive.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[890235f](https://github.com/OfficeDev/office-scripts-docs/commits/890235fec07961024509e06b56d110d0c348b670)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Issue_Template.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/Issue_Template.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/README.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/index.md) | :white_check_mark:Succeeded | |
[docs/overview/code-editor-environment.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/code-editor-environment.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/testing/troubleshooting.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/testing/troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131943288472-56%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Dw5GEdvNXMPivwuyHqNQ7N%2F8EEHqSq%2BNPKBTVgh4hUA%3D&st=2019-12-13T19%3A39%3A40Z&se=2020-01-13T19%3A44%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[42c6d41](https://github.com/OfficeDev/office-scripts-docs/commits/42c6d4137bd09738fe213b4fe297ed59d1e54f41)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Issue_Template.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/Issue_Template.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/README.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/index.md) | :white_check_mark:Succeeded | |
[docs/overview/code-editor-environment.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/code-editor-environment.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/testing/troubleshooting.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Intros/docs/testing/troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131944228229-56%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0Cs3EG7FmSbY%2BpuxJlGbOTlh%2BRQ0goIqwKG26rg6Kf0%3D&st=2019-12-13T19%3A40%3A51Z&se=2020-01-13T19%3A45%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**